src/sentry/uptime/migrations/0033_uptime_backfill_to_detectors.py

```diff
@@ -0,0 +1,89 @@
+# Generated by Django 5.1.7 on 2025-04-17 20:20
+
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.uptime.models import UptimeSubscription as UptimeSubscriptionType
```

Comment:
Shouldn't import this here, it's incorrect anyway since it accepts the fake type

Reply:
mypy was complaining if I don't type my function

Reply:
Hmm, the types here are wrong though. They're dynamically generated by the migration system. Not really sure how to work around this, but I'd prefer to just use `Any` instead of importing the external types

Reply:
All good just gonna change this fucntion to `has_detector(sub_id: int) → bool`